library/src/main/java/com/novoda/downloadmanager/lib/RequestBatch.java

```diff
@@ -0,0 +1,50 @@
+package com.novoda.downloadmanager.lib;
+
+import android.content.ContentValues;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RequestBatch {
+
+    private final BatchInfo batchInfo;
+    private final List<Request> requests;
+
+    public static RequestBatch newInstance(String title, String description, String bigPictureUrl) {
```

Comment:
no that's true they should be optional, not everyone wants a notification. Therefore when optional the builder pattern works (which is what the other `Request` object adhere's to. Also allows us to add more params more extensibly than this does.)